osu.Game.Rulesets.Taiko/Skinning/Legacy/LegacyDrumRoll.cs

```diff
@@ -47,13 +48,13 @@ private void load(ISkinSource skin, OsuColour colours)
                     Anchor = Anchor.CentreRight,
                     Origin = Anchor.CentreLeft,
                     RelativeSizeAxes = Axes.Both,
-                    Texture = skin.GetTexture("taiko-roll-end", WrapMode.ClampToEdge, WrapMode.ClampToEdge),
+                    Texture = skin.GetTexture("taiko-roll-end", new Vector2(128, 256), WrapMode.ClampToEdge, WrapMode.ClampToEdge),
```

Comment:
I intentionally haven't included osu!taiko drum rolls because they're already being resized in the `Sprite` definition (notice RSA = Both & FillMode = Fit).